script.js

```diff
@@ -18,4 +18,233 @@ mobileLinks.forEach((link) => {
+                    <img data-close-button class ="close" src="./Images/Icon - Cancel.png" alt="popup close icon">
+                    </div>
+                    <div class="desktop-popup-buttons">
+                        <button class="desk-btn-1">
+                          <div class="popup-btn-text">${cardData[0].btn1}</div>
+                          <div class="popup-icons">
+                            <img src="./Images/see live icon.png" alt="see-live-icon">
+                          </div>
+                        </button>
+                        <button class="desk-btn-2">
+                        <div class="popup-btn-text">${cardData[0].btn2}</div>
+
+                          <div class="popup-icons">
+                            <img src="./Images/Icon -GitHub.png" alt="see-live-icon">
+                          </div>
+                        </button>
+                      </div>
+                    <div class="project-popup-title">
+                    <h3>${cardData[0].popTitle}</h3>
+                    <ul class="popup-project-abilites2">
+                          <li class="a1">${cardData[0].popTechnologies2[0]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[1]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[2]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[3]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[4]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[5]}</li>
+                      </ul>
+
+                    <ul class="popup-project-abilites">
+                        <li class="a1">${cardData[0].popTechnologies1[0]}</li>
+                        <li class="a1">${cardData[0].popTechnologies1[1]}</li>
+                        <li class="a1">${cardData[0].popTechnologies1[2]}</li>
+                    </ul> 
+                    </div>
+                    
+                    <p class="mobile-popup-text">
+                        ${cardData[0].capstoneIntro1}
+                    </p>
+                    <div class="mobile-popup-buttons">
+                    <a href=${cardData[0].linkToLiveSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${cardData[0].btn1}</div>
+                        <div class="popup-icons">
+                        <img src="./Images/see live icon.png" alt="see-live-icon">
+                        </div>
+                    </button>
+                    </a>
+                    <a href=${cardData[0].linkToSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${cardData[0].btn2}</div>
+                        <div class="popup-icons">
+                        <img src="./Images/Icon -GitHub.png" alt="see-live-icon">
+                        </div>
+                    </button>
```

Comment:
- [ ] Please use all button effects mentioned in the Figma design you are not using button effects on your `popup` buttons. Check below :point_down: the comparison between your design and Figma design :+1:

| Your Design   |      Figma Design      |
|---------- |:----------------:|
| ![Screenshot from 2023-04-19 00-53-49](https://user-images.githubusercontent.com/45226848/232893972-d2291a44-dcd6-467a-9f24-2e65d57b6215.png) |  ![Screenshot from 2023-04-19 00-54-16](https://user-images.githubusercontent.com/45226848/232894027-d10b0024-07eb-414b-8a4e-de36817f99f8.png)    |  

- [ ] Also, the `width` of your buttons are not matched the Figma design Kindly fix it :slightly_smiling_face: